index.html

```diff
@@ -543,6 +543,111 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="production">
+      <h2>Production</h2>
+
+      <p>
+        This section describes how to produce a VC-JWT encoded
+        <code>VerifiableCredential</code> from a <code>Credential</code>.
+      </p>
+
+      <p class="advisment">
+        There are currently 2 competing solutions to this problem described below. It is a goal
+        of the v2 work to resolve them and
+        reduce production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they possess a software library
+        supporting [[RFC7515]] or [[RFC7519]].
+      </p>
+
+      <section>
+        <h3>In addition to...</h3>
+
+        <p>
+          There are several members (claims) of the
+          <code>Credential</code> which will need to be translated to their JOSE
+          form, and included next to the <code>vc</code> or
+          <code>vp</code> member in the <code>payload</code>.
+        </p>
+
+        <p>
+          If a member is not present in the <code>Credential</code> it MUST NOT
+          be present in the <code>VerifiableCredential</code> as either a claim
+          in the payload or a claim in the <code>vc</code> attribute of the
+          payload.
+        </p>
+
+        <p>
+          We start with an empty header, and payload objects, and we add members
+          to the header and the payload based on the content in the
+          <code>Credential</code>.
+        </p>
+
+        <section>
+          <h4>issuer -> iss</h4>
+
+          <p>
+            This member MUST be present in the
+            <code>payload.vc.issuer</code> attribute as either a string or an
+            <code>object</code> with and <code>id</code>.
+          </p>
+
+          <p>This member MUST be present <code>payload.iss</code>.</p>
+
+          <p>
+            In the case that <code>payload.vc.issuer</code> is an object,
+            <code>payload.iss</code> MUST be <code>payload.vc.issuer.id</code>.
+          </p>
+
+          <p>
+            In the case that
+            <code>payload.vc.issuer</code> is a string,
+            <code>payload.iss</code> must be
+            <code>payload.vc.issuer</code>
+          </p>
+        </section>
+
+        <section>
+          <h4>issuanceDate -> nbf</h4>
```

Comment:
Nope.  issuanceDate -> iss.

Reply:
See also the current spec text: 
![Screen Shot 2022-10-21 at 8 27 31 AM](https://user-images.githubusercontent.com/8295856/197207064-8f39db87-c8bb-4a87-9dcd-7c76223a4970.png)

Reply:
I agree that given the current W3C DM definition, then issuanceDate maps to nbf

Reply:
However, I also think that the spec should allow the signer to optionally add iat as well, especially if the nbf is not equal to "now", which iat will always be.

Reply:
I prefer we take that conversation to issues, the intention of this PR is not to change anything, only to make what is currently legal, clearer.

Reply:
Per the consensus call on last week's call, I believe that issuanceDate is going to become validFrom.  I'm fine with validFrom being mapped to `nbf`.

Reply:
Filed, https://github.com/w3c/vc-jwt/issues/38 We will not forget to do this.